packages/modal/addon/components/nucleus-modal/dialog.js

```diff
@@ -180,6 +188,52 @@ class Dialog extends Component {
     }
   }
 
+  /**
+  * createFocus
+  *
+  * @method createFocus
+  * @private
+  *
+  */
+  _createFocus() {
+    let modalDialog = this.element;
+    let focusEl = modalDialog && modalDialog.querySelector("[autofocus]");
+    if(focusEl) {
+      focusEl.focus();
+    }
+    else {
+      modalDialog.focus();
+    }
+  }
+
+  /**
+  * loopFocus
+  *
+  * @method loopFocus
+  * @private
+  * @param {any} event
+  */
+  loopFocus(event) {
+    let isTab = (event.key === 'Tab' || event.keyCode === 9);
+    if (!isTab) {
+      return;
+    }
+    let modalDialog = this.element.querySelector(".nucleus-modal__dialog");
+    let focusElements = [...modalDialog.querySelectorAll('a[href]:not([disabled]), button:not([disabled]), textarea:not([disabled]), input[type="text"]:not([disabled]), input[type="radio"]:not([disabled]), input[type="checkbox"]:not([disabled]), select:not([disabled])')];    
```

Comment:
Lets move the allowed focusable elements string as a const

Reply:
Okay